pkg/kubectl/cmd/logs.go

```diff
@@ -94,9 +100,7 @@ func NewCmdLogs(f cmdutil.Factory, out io.Writer) *cobra.Command {
 		},
 		Run: func(cmd *cobra.Command, args []string) {
 			cmdutil.CheckErr(o.Complete(f, out, cmd, args))
-			if err := o.Validate(); err != nil {
-				cmdutil.CheckErr(cmdutil.UsageError(cmd, err.Error()))
-			}
+			cmdutil.CheckErr(o.Validate())
```

Comment:
All user input is verified in `Complete`, at this point we're validating the correctness of internal structures (here `LogsOptions`) before running the actual command. User has no real input on these values, thus this doesn't provide any value for him. See other commands for similarity.